packages/sw-routing/test/browser/register-unit-tests.js

```diff
@@ -1,11 +1,11 @@
-describe('Service Worker Unit Tests', function() {
+describe('Service Worker Unit Test Registration', function() {
   beforeEach(() => window.goog.swUtils.cleanState());
 
   const swUnitTests = [
-    'sw-unit/namespace.js',
-    'sw-unit/route.js',
-    'sw-unit/express-route.js',
-    'sw-unit/regexp-route.js',
+    'unit/namespace.js',
```

Comment:
Pro the renaming (See above) would like to just standardise and implement it across the tests (Mine are due a refactor).